frontend/src/tests/user-permission-tests.js

```diff
@@ -14,11 +14,43 @@ import { databaseSeeder } from "./setup";
 export function userPermissionsTests(api) {
     const { apiGET, apiPOST } = api;
 
-    const newUserData = {
+    const newInstructorData = {
         utorid: "userxx",
         roles: ["instructor"],
     };
 
+    const newTAData = {
+        utorid: "tauser",
+        roles: ["ta"],
+    };
+
+    const newTAInstructorData = {
+        utorid: "tainstuser",
+        roles: ["ta", "instructor"],
+    };
+
+    async function setupUserPerm(userData) {
+        let respPrevUser = await apiGET("/active_user");
```

Comment:
Maybe I am wrong, but I believe since respPrevUser is not changed we can use const here, as what is prevUserId